project/backend/user-service/docker-compose.yml

```
@@ -0,0 +1,28 @@
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker # Keep this if using profiles
+
+      # OTel Configuration
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4318
+      - OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+      - OTEL_SERVICE_NAME=user-service
+
+      # === CORRECTED Database connection variables ===
+      # Use standard Spring Boot environment variable names
+      - SPRING_DATASOURCE_URL=jdbc:mysql://host.docker.internal:3306/metadocs
+      - SPRING_DATASOURCE_USERNAME=root # <<< CHANGE
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)